internal/test/container_test.go

```diff
@@ -108,6 +108,43 @@ func Test_configureSession(t *testing.T) {
 	})
 }
 
+func Test_loadImage(t *testing.T) {
+	absoluteTileDirectory := filepath.Join(t.TempDir(), "test")
+	logger := log.New(io.Discard, "", 0)
+	t.Run("when loading a provided test image with a wrong path", func(t *testing.T) {
+		ctx := context.Background()
+		out := bytes.Buffer{}
+		configuration := Configuration{
+			AbsoluteTileDirectory: absoluteTileDirectory,
+			ImagePath:             "non-existing",
+		}
+		client := runTestWithSessionHelper(t, "", container.WaitResponse{
+			StatusCode: 0,
+		})
+
+		err := runTestWithSession(ctx, logger, &out, client, configuration)("some-session-id")
+		require.ErrorContains(t, err, "failed to read image 'non-existing': open non-existing: no such file or directory")
+
+	})
+	t.Run(`when loading a provided test image with an existing path`, func(t *testing.T) {
+		ctx := context.Background()
+		out := bytes.Buffer{}
+
+		configuration := Configuration{
+			AbsoluteTileDirectory: absoluteTileDirectory,
+			ImagePath:             `assets/alpine.tgz`,
+		}
+
+		client := runTestWithSessionHelper(t, "", container.WaitResponse{
+			StatusCode: 0,
+		})
+
+		err := runTestWithSession(ctx, logger, &out, client, configuration)("some-session-id")
+		require.NoError(t, err)
+
+	})
+}
+
```

Comment:
The test looks good but I am not able to run it on my machine (again). Not sure why but it seems like a docker issue on my machine.